use-timescale/schema-management/foreign-data-wrappers.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Foreign data wrappers
+excerpt: Query PostgreSQL databases within or outside Timescale
+products: [cloud, mst, self_hosted]
+keywords: [hypertables, schemas, alter]
+tags: [change]
+---
+
+# Foreign data wrappers
+
+You use foreign data wrappers (FDWs) to query external data sources from a Timescale Cloud service. These data sources can be one of the following:
+
+- Other Timescale Cloud services.
+- PostgreSQL databases outside of Timescale Cloud.
+
+If you are using [VPC peering][vpc-peering], you can create FDWs in your Customer VPC for a service in Timescale VPC or the same project. However, you can't create FDWs from Timescale VPC to Customer VPC.
+
+FDWs are particularly useful if you manage multiple different Timescale Cloud service types, and need to seamlessly access and merge regular and time-series data.
+
+## Query another data source
+
+You create FDWs with the help of the `postgres_fdw` extension, which is enabled by default. 
+
+<Procedure>
+
+To query another data source, run the following queries in the [SQL editor][sql-editor]:
+
+1. Create a server:
+
+   ```sql
+   CREATE SERVER <server-name> FOREIGN DATA WRAPPER postgres_fdw OPTIONS (host '<service-ID>.<project-id>.tsdb.cloud.timescale.com', dbname '<database-name>', port '<port-number>');
+   ```
+
+1. Create user mapping:
+
+   ```sql
+   CREATE USER MAPPING FOR <tsdbadmin> SERVER <server-name> OPTIONS (user '<tsdbadmin>', password '<tsdbadmin-password>');
+   ```
+
+1. Import a foreign schema (recommended) or create a foreign table:
+
+    - Import the whole schema:
+
+      ```sql
+      CREATE SCHEMA <schema-name>;
+      IMPORT FOREIGN SCHEMA <foreign-schema-name> FROM SERVER <server-name> INTO <schema-name>;
+      ```
+      
+    - Alternatively, import a limited number of tables: 
+
+      ```sql
+      CREATE SCHEMA <schema-name>;
+      IMPORT FOREIGN SCHEMA <foreign-schema-name> LIMIT TO (table1, table2) FROM SERVER <server-name> INTO <schema-name>;
+      ```
+
+    - Create a foreign table: 
+
+      ```sql
+      CREATE FOREIGN TABLE <table-name> (
+      code        char(5) NOT NULL,
```

Comment:
Assuming a user has a table defined like this:

```
create table metric(recorded timestamptz not null, temp float, device text);
```

A corresponding foreign table will look like:

```
CREATE FOREIGN TABLE metric (
recorded timestamptz NOT NULL,
temp float,
device text
) SERVER <server-name>;
```

I would still prefer to explicitly mention it's an alternative to import, because a lot of users just copy all commands to console directly without contemplating :smiling_face_with_tear:

Reply:
@inchuev This is more of an example, and all other commands use placeholders. Would it be correct to modify like this:

```
CREATE FOREIGN TABLE <table-name> (
<column-1-name> <data-type> <optional-constraint>,
...
<column-n-name> <data-type> <optional-constraint>
) SERVER <server-name>;
```

?

Reply:
Yes, this is correct.